README.md

```diff
@@ -68,4 +68,11 @@ Notes and Copyright
 This tool is inspired by [redis-faina](https://github.com/Instagram/redis-faina).
 The tool was forked from [solr-loganalyzer](https://github.com/dfdesho/solr-loganalyzer).
 
-This tool is distributed with the [BSD-2 Licence](LICENCE).
\ No newline at end of file
+This tool is distributed with the [BSD-2 Licence](LICENCE).
+
+Authors
+=======
+
+* Martin Filliau <dfdeshom@gmail.com>
```

Comment:
My name is Didier Deshommes :)

Reply:
I am sorry, i accidentally picked up the wrong name :-)